snipeit.sh

```diff
@@ -247,58 +245,77 @@ case $distro in
         perms
         service apache2 restart
         php artisan key:generate
-        php artisan passport:install
         ;;
     ubuntu)
         #####################################  Install for Ubuntu  ##############################################
-        #Update/upgrade Debian/Ubuntu repositories, get the latest version of git.
-        #Git clone snipeit, create vhost, .env file, mysql install
-        #composer install, set permissions, restart apache.
 
         webdir=/var/www
-        echo -ne "\n* Adding MariaDB repo in the background... ${spin[0]}"
+
+        echo "* Adding MariaDB repository."
         (echo "deb [arch=amd64,i386] http://ftp.hosteurope.de/mirror/mariadb.org/repo/10.1/ubuntu $codename main" | tee /etc/apt/sources.list.d/mariadb.list >/dev/null 2>&1)
         log "apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0xF1656F24C74CD1D8"
-        echo -ne "\n* Updating with apt-get update in the background... ${spin[0]}"
+
+        echo "* Updating with apt-get update."
         log "apt-get update" & pid=$!
         [ -f /var/lib/dpkg/lock ] && rm -f /var/lib/dpkg/lock
         progress
-        echo -ne "\n* Upgrading packages with apt-get upgrade in the background... ${spin[0]}"
+
+        echo "* Upgrading packages with apt-get upgrade."
         log "apt-get -y upgrade" & pid=$!
         progress
-        echo -ne "\n* Setting up LAMP in the background... ${spin[0]}\n"
-        log "DEBIAN_FRONTEND=noninteractive apt-get install -y mariadb-server mariadb-client apache2 libapache2-mod-php curl" & pid=$!
+
+        echo "* Installing httpd, PHP, MariaDB and other requirements."
+        log "DEBIAN_FRONTEND=noninteractive apt-get install -y mariadb-server mariadb-client apache2 libapache2-mod-php curl git unzip" & pid=$!
         progress
+
         if [ "$version" == "16.04" ]; then
-            log "apt-get install -y git unzip php php-mcrypt php-curl php-mysql php-gd php-ldap php-zip php-mbstring php-xml" & pid=$!
```

Comment:
Does this need to be expanded to work on 16.10 and 17.04 ? I'm guessing the packages are named "php" on most everything now.. Can't remember the history of this.

Reply:
Absolutely. I was having trouble downloading the 17.04 iso. So i will look into this later today.

Still have a bit to go.